lib/encodings.js

```diff
@@ -3,92 +3,92 @@
 const errors = require('http-errors')
 const zlib = require('zlib')
 
-module.exports = Encodings
+class Encodings {
```

Comment:
sorry, there's too much going on here. Better to make more smaller PRs

Reply:
Done, this is smaller #137 and it was taken all latest updates in consideration.